test/server/cards/02_SHD/events/LookTheOtherWay.spec.ts

```diff
@@ -0,0 +1,132 @@
+                expect(context.player2.countExhaustedResources()).toBe(2);
+            });
+
+            it('should does not exhaust unit because controller pays 2 resources (friendly)', function () {
+                const { context } = contextRef;
+
+                context.player1.clickCard(context.lookTheOtherWay);
+                expect(context.player1).toBeAbleToSelectExactly([context.viperProbeDroid, context.wampa]);
+                context.player1.clickCard(context.wampa);
+
+                expect(context.player1).toHaveEnabledPromptButtons(['Exhaust Wampa', 'Pay 2 resources']);
+                context.player1.clickPrompt('Pay 2 resources');
+                expect(context.wampa.exhausted).toBeFalse();
+                expect(context.player1.countExhaustedResources()).toBe(2);
+            });
+        });
+
+        describe('Look the Other Way\'s ability', function () {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['look-the-other-way'],
+                        groundArena: ['wampa'],
+                        base: 'jabbas-palace'
+                    },
+                    player2: {
+                        groundArena: ['viper-probe-droid'],
+                        resources: 1,
+                    }
+                });
+            });
+
+            it('should exhaust an enemy unit without choice because opponent does not have sufficient resources', function () {
+                const { context } = contextRef;
+
+                context.player1.clickCard(context.lookTheOtherWay);
+                expect(context.player1).toBeAbleToSelectExactly([context.viperProbeDroid, context.wampa]);
+                context.player1.clickCard(context.viperProbeDroid);
+                expect(context.viperProbeDroid.exhausted).toBeTrue();
+                expect(context.player2).toHavePrompt('Choose an action');
+            });
+        });
+
+        describe('Look the Other Way\'s ability', function () {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['look-the-other-way'],
+                        groundArena: ['wampa'],
+                        base: 'jabbas-palace'
+                    },
+                    player2: {
+                        hand: ['battlefield-marine'],
+                        groundArena: ['viper-probe-droid'],
+                        base: 'echo-base',
+                        leader: 'sabine-wren#galvanized-revolutionary',
+                        hasInitiative: true,
+                        resources: 3,
+                    }
+                });
+            });
+
+            it('should exhaust an enemy unit without choice because opponent does not have sufficient ready resources', function () {
```

Comment:
Not sure what makes this test different than the one above, can you clarify?

Reply:
I don't know if there is a difference between do not have resource to pay (total resource) and do have total resource to pay but some are exhausted

Reply:
Oh nah not really, I don't think there's enough of a distinction to be worth having separate tests for. If that was a problem then like 100 other tests would break at least